### PR TITLE
linters added

### PR DESCRIPTION
**Pull request by dhy-612**
Milestone1

Two Sections I have created:
The Toolbar (or header).
The headline section (right after the header).

I have used Flexbox in all 2 sections.
I have made the basic structure of the project i.e. Header, Headline using Figma templates.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @dhy-612 👋, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Your pull request summary is not descriptive. Kindly add a descriptive pull request summary
- [x] You don't have a README.md file. Kindly use Microverse's README template. Please remember to populate information for all headers in the REDAME.md file. The optional header is the FAQ section. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -4,9 +4,56 @@
...
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <header>
+        <nav>
+            <div class="Navigation">
+                <a href="#" class="logo">HARSH</a>
+                <a href="#" class="menu"><img src="./images/menu.svg" alt="menu"></a>
+            </div>
```
Comment (joskalenda) [inline index.html:16]: - [ ] Your design is awesome @dhy-612, congratulations, however, I have a few points to mention regarding the code syntax, as per the requirement, it is asked to have 2 sections, the tagline`(header{logo and nav bar})` and the headline`(section body)`. this to say, you should not warp the entire app in the header tag, this is not good sematicaly either, please do something like this

```
<header>
    logo a
    <nav>
       nav item
    <nav>  
<header>
<section>
    section content
<section>

```

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @dhy-612 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Good design
- [ ].. ✅ Great readme file
- [ ].. ✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @dhy-612 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
GitHub Flow observed✔️
Linters Set up correctly ✔️ 
App worked as expected ✔️ 
Good use of HTML semantics element ✔️ 

## Optional suggestions

- [optional] Please kindly ensure that your Pull Request have a descriptive title and descriptive branch name telling us what changes are inside. ℹ️  

- [optional] Please kindly update the README file with the `Live Demo` and `Future Features` headers. You can add a working `live demo link` if you have one, if not I think you can indicate **NOT AVAILABLE AT THE MOMENT**.  All these will make your repo looks professional. 👍 

- [optional] Commit messages are suppose to be descriptive; in imperative tense and capitalized, please take note of this for subsequent PRs. [Reference](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) Thanks. ✋   

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**